### PR TITLE
Create maven settings to deploy snapshots to oss.jfrog.org

### PR DESCRIPTION
**Pull request by dacrome**

### CODE REVIEW

File: settings.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <server>
+            <id>bintray</id>
+            <username>${env.BINTRAY_USER}</username>
+            <password>${env.BINTRAY_KEY}</password>
+        </server>
+    </servers>
+    <profiles>
+        <profile>
+            <id>oss-jfrog-snapshots</id>
+            <repositories>
```
Comment (tkrille) [inline settings.xml]: Shouldn't this go into our POMs?

Comment (dacrome) [inline settings.xml]: No, but a [distributionManagment](https://github.com/osiam/connector4java/pull/198/files#diff-600376dffeb79835ede4a0b285078036R68)

Comment (tkrille) [inline settings.xml]: But I think, that at least the add-ons and the ITs need this, like they needed the Evolvis repository before.

Comment (dacrome) [inline settings.xml]: Yes, like I did in the [addon-administration](https://github.com/osiam/addon-administration/pull/130/files#diff-600376dffeb79835ede4a0b285078036R223)

Comment (tkrille) [inline settings.xml]: But if each project, which needs access to snapshots (probably just the ITs and add-ons), already has this `repository` block in their POM, why do we need it here again?

Comment (tkrille) [inline settings.xml]: I mean, if this is needed to build OSIAM, then it does not belong here, but only in the POMs, does it?

Comment (dacrome) [inline settings.xml]: This is how it is documented to use the settings.xml with the target repository and the active profile. The repository within the pom is for snapshot dependencies... Hm, but maybe you are right and the target repository is set with the distributionManagment and do not need to be set twice... I will investigate in how to use the settings.xml.

Comment (dacrome) [inline settings.xml]: Hey @tkrille. You are absolutely right. I do not need to specify a profile with a repository. I only need to set a server with username and password. Tested with a upload of a snapshot of the connector4java.

**Comment by tkrille** [comment]
Alright, LGTM. Thanks!

**Pull request merged**